CHANGELOG.md

```diff
@@ -147,6 +157,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2501](https://github.com/Pycord-Development/pycord/pull/2501))
 - Deprecated `Interaction.cached_channel` in favor of `Interaction.channel`.
   ([#2658](https://github.com/Pycord-Development/pycord/pull/2658))
+- Deprecated `nsfw` for categories, as it was never supported.
```

Comment:
```suggestion
- Deprecated `is_nsfw` for categories, as it was never supported.
```

Reply:
The nsfw attribute itself is not technically deprecated in the code. For it to be deprecated, it should be made into a property and then marked with `@deprecated`

Reply:
```suggestion
- Deprecated `is_nsfw` for categories since it was never supported by the API.
```